Cannot find module 'fs-promise'

I have a problem
```bash              
> portfolio@1.0.0 start /media/id/hdd/dev/sites/portfolio
> gulp

[00:06:54] Requiring external module babel-core/register
module.js:341
    throw err;
    ^

Error: Cannot find module 'fs-promise'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/media/id/hdd/dev/sites/portfolio/node_modules/postcss-svg-fragments/index.js:1:77)
    at Module._compile (module.js:413:34)
    at Module._extensions..js (module.js:422:10)
    at Object.require.extensions.(anonymous function) [as .js] (/media/id/hdd/dev/sites/portfolio/node_modules/babel-register/lib/node.js:138:7)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)


```
This happens because of the warning during installation?
```bash
➜  portfolio git:(master) ✗ npm i -D postcss-svg-fragments
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.7
npm WARN portfolio@1.0.0 No description
npm WARN portfolio@1.0.0 No repository field.

```

I try delete node_modules and install doesn't fix the problem

```bash
➜ node -v && npm -v
v5.5.0
v3.7.1
```
System: Mint 17.2

--- a/package.json
+++ b/package.json
@@ -4,13 +4,13 @@
   "description": "Use SVG fragments in CSS URLs",
   "main": "index.js",
   "dependencies": {
+    "fs-promise": "^0.4.1",
     "postcss": "^5.0.11",
     "postcss-value-parser": "^3.1.1",
     "xmldoc": "^0.3.2"
   },
   "devDependencies": {
     "eslint": "^1.9.0",
-    "fs-promise": "^0.4.1",
     "tap-spec": "^4.1.0",
     "tape": "^4.2.2"
   },

